PowerShellTools.Test/IntelliSense/CommentAreaUnitTests.cs

```diff
@@ -73,7 +73,7 @@ public void TestNormalCommentScript()
             IsInCommentAreaTestHelper(script, 10, true);
             IsInCommentAreaTestHelper(script, 100, false);
             IsInCommentAreaTestHelper(script, 300, true);
-            IsInCommentAreaTestHelper(script, script.Length, true);
+            IsInCommentAreaTestHelper(script, script.Length - 1, true);
         }
 

```

Comment:
No, there are a bunch of tests needed for IntelliSense and other related stuff. I suggest revisiting this when we have more time.